schema/opat.schema.json

```diff
@@ -1,14 +1,91 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://github.com/GSA/open-product-accessibility-template/schema/opat.schema.json",
+  "$id": "opat.schema.json",
   "title": "OPAT",
-  "description": "Open Product Accessibility Template (OPAT)",
+  "description": "Open Product Accessibility Template (OPAT) as a YAML file.",
   "type": "object",
   "properties": {
     "title": {
-      "description": "The title of the report in the heading format of '[Company Name] Accessibility Conformance Report'",
+      "description": "The title of the report in the heading format of '[Company Name] Accessibility Conformance Report'.",
       "type": "string"
+    },
+    "name-product-version": {
+      "description": "Product name and version if available.",
+      "type": "string"
+    },
+    "product-description": {
+      "description": "A brief description of the product.",
+      "type": "string"
+    },
+    "contact-information": {
+      "description": "Primary contact email.",
+      "type": "string"
+    },
+    "notes": {
+      "description": "Any details or explanation about the product or the report.",
+      "type": "string"
+    },
+    "chapters": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "num": {
+            "description": "Table or chapter number.",
+            "type": "string"
+          },
+          "criteria": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "num": {
+                  "description": "Criteria number.",
+                  "type": "string"
+                },
+                "components": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "description": "Name of component. E.g. Web, Electronic Docs, Authoring Tools, Software, and so on.",
```

Comment:
I think these would be defined in the catalog and would be part of the validation outside of JSON Schema? Might be good to establish a convention of describing where/how the schema references a field that is managed in the catalog?

Reply:
Yes, I agree about the catalog containing the names. I think the catalog ticket #10 we can tackle how to reference them in a consistent manner.